boards/arm/adafruit_itsybitsy_nrf52840/adafruit_itsybitsy_nrf52840.yaml

```
@@ -0,0 +1,22 @@
+identifier: adafruit_itsybitsy_nrf52840
+name: Adafruit ItsyBitsy nRF52840 Express
+type: mcu
+arch: arm
+ram: 256
+flash: 1024
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - adc
+  - ble
+  - counter
+  - gpio
+  - i2c
+  - ieee802154
```

```suggestion
  - ieee802154
```

this is obsolete, please delete